### PR TITLE
[CDAP-3838] adding last finished info on hydrator detail status tab

### PR DESCRIPTION
**Pull request by elfenheart**
screen shot 2015-10-06 at 11 14 00 am

screen shot 2015-10-06 at 11 14 12 am

### CODE REVIEW

File: cdap-ui/app/features/hydrator/controllers/tabs/status-ctrl.js [hunk 1]
```diff
@@ -24,15 +24,22 @@ angular.module(PKG.name + '.feature.hydrator')
...
 
-    HydratorDetail.api.runs(params)
+    HydratorDetail.api.pollRuns(params)
       .$promise
       .then(function (runs) {
         $scope.stats.numRuns = runs.length;
         $scope.stats.lastRunTime = runs.length > 0 && runs[0].end ? runs[0].end - runs[0].start : 'N/A';
+
+        for (var i = 0; i < runs.length; i++) {
+          if (runs[i].status !== 'RUNNING') {
```
Comment (anwar6953) [inline cdap-ui/app/features/hydrator/controllers/tabs/status-ctrl.js]: A potential status is 'STARTING', which is not finished, but this code would assume it is finished, right?

**Comment by ajainarayanan** [comment]
@anwar6953 I think we want to identify the last "Killed" or "Completed" or "Stopped" run. So we are navigating through the list of runs to find the latest run that is having either of these states (or not one of 'Starting' or 'Stopping' or 'Running').

**Comment by ajainarayanan** [comment]
Same as the other PR (for 3.2) If for one comment :+1: LGTM. (if @anwar6953 is convinced :smile: )

**Comment by anwar6953** [comment]
I see. I thought there was only one state it would check against - 'STOPPED'.
Since there's also KILLED and COMPLETED, I guess its significantly better.

LGTM.

**Pull request merged**